src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -91,6 +95,55 @@ private enum Booleans : uint
             IsWebSocketRequest = 0x00000800,
             Default = AllowAutoRedirect | AllowWriteStreamBuffering | ExpectContinue
         }
+
+        private class HttpClientParameters
+        {
+            public readonly DecompressionMethods AutomaticDecompression;
+
+            public readonly bool AllowAutoRedirect;
+
+            public readonly int MaximumAutomaticRedirections;
+
+            public readonly int MaximumResponseHeadersLength;
+
+            public readonly bool PreAuthenticate;
+
+            public readonly int Timeout;
+
+            public readonly SslProtocols SslProtocols;
+
+            public readonly bool CheckCertificateRevocationList;
+
+            public HttpClientParameters(HttpWebRequest webRequest)
+            {
+                AutomaticDecompression = webRequest.AutomaticDecompression;
+                AllowAutoRedirect = webRequest.AllowAutoRedirect;
+                MaximumAutomaticRedirections = webRequest.MaximumAutomaticRedirections;
+                MaximumResponseHeadersLength = webRequest.MaximumResponseHeadersLength;
+                PreAuthenticate = webRequest.PreAuthenticate;
+                Timeout = webRequest.Timeout;
+                SslProtocols = (SslProtocols)ServicePointManager.SecurityProtocol;
```

Comment:
Why cast back and forth from SecurityProtocolType to SslProtocols rather than just using SecurityProtocolType?

Reply:
It's a mistake. Initially, I assumed that SslProtocols is a HttpWebRequest's property, but it actually is retrieved from ServicePointManager every time HttpClient is created.
Fixed.